.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: node_js
 node_js:
   - "6"
-  - "5"
   - "4"
+
```

Comment:
Any reasons for this whitespace? If not, I'd suggest to remove it as well to stay on the context.